README.md

```diff
@@ -1,29 +1,29 @@
-# ZeppelinOS library
+# ZeppelinOS library _(zos-lib)_
+
 [![NPM Package](https://img.shields.io/npm/v/zos-lib.svg?style=flat-square)](https://www.npmjs.org/package/zos-lib)
+
 [![Build Status](https://travis-ci.org/zeppelinos/zos-lib.svg?branch=master)](https://travis-ci.org/zeppelinos/zos-lib)
+
 [![Coverage Status](https://coveralls.io/repos/github/zeppelinos/zos-lib/badge.svg?branch=master)](https://coveralls.io/github/zeppelinos/zos-lib?branch=master)
 
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
 :warning: **Under heavy development: do not use in production** :warning:
 
-`zos-lib` is a library for writing upgradeable smart contracts on Ethereum. It can be used to create an upgradeable on-chain distributed application.
+> Library for writing upgradeable smart contracts on Ethereum.
 
-Use this library if you want to programmatically develop, deploy or operate an upgradeable smart contract system.
+Use `zos-lib` if you want to programmatically develop, deploy or operate an upgradeable smart contract system.
 
-If you want a CLI-aided development experience, see [the zOS CLI](https://github.com/zeppelinos/zos-cli).
+We recommend the CLI-aided development experience instead of this lower level library. See [the zOS CLI](https://github.com/zeppelinos/zos-cli).
 
-# Getting Started
+## Install
 
 To install `zos-lib` simply go to your project's root directory and run:
 ```sh
-npm i zos-lib
+npm install zos-lib
 ```
 
-Next, learn how to:
-- [Develop and deploy a single smart contract which can be upgraded](#single) (for bugfixing or adding new features).
-- [Develop and operate a complex upgradeable app](#complex) with multiple smart contracts which are connected to the zOS upgradeable standard libraries.
-
-## <a name="single"></a> Develop and deploy a single upgradeable smart contract
-Note: This shows a low-level manual method of developing a single upgradeable smart contract. You probably want to use [the higher-level CLI guide](https://github.com/zeppelinos/zos-cli/blob/master/README.md).
+## Usage
 
 To work with a single upgradeable smart contract, we just need to deal with a simple upgradeability proxy. This is a special contract that will hold the storage of our upgradeable contract and redirect function calls to an `implementation` contract, which we can change (thus making it upgradeable). Let's do the following example to see how it works:
```

Comment:
Following the README refactor goals (and for consistency with the changes introduced in this PR), we should relocate this guide too.